### PR TITLE
Rules, Questions and Result Pages

### PR DESCRIPTION
**Pull request by AbhishekkGautam**
### Rules Page

rules-page

### Questions Page

questions-page

### Results Page

result-page

### CODE REVIEW

File: pages/result/result.css [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+.container {
+  margin-bottom: 6rem;
+}
+
+.section-wrapper {
+  min-height: 80vh;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  justify-content: center;
```
Comment (msdeshmukh009) [inline pages/result/result.css:13]: Consider making utility class for these styles as you are using them oftenly. 
```html
  display: flex;
  align-items: center;
  justify-content: center;
```

**Review by msdeshmukh009 — APPROVED** [review]
The rest is looking good.

File: pages/result/result.css (hunk 1 above)
Comment (AbhishekkGautam) [reply pages/result/result.css:13]: Okay. will do that.

File: pages/questions/questions.css [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  font-size: 1.5rem;
+  text-align: center;
+}
+
+.questions-card-container .card-body .top-display {
+  display: flex;
+  justify-content: space-between;
+}
+
+.questions-card-container .card-body .top-display .count .text {
```
Comment (kiranpratyush) [inline pages/questions/questions.css:35]: we can give a class Name to the element. It might create specificity issues that is had to figure out

**Review by kiranpratyush — APPROVED** [review]
We can format HTML files for better looking

File: pages/questions/questions.css (hunk 2 above)
Comment (AbhishekkGautam) [reply pages/questions/questions.css:35]: Yes, these all are separate class names given to the nested elements

**Comment by AbhishekkGautam** [comment]
> We can format HTML files for better looking

I think it's already formatted.

File: pages/questions/questions.css (hunk 2 above)
Comment (kiranpratyush) [reply pages/questions/questions.css:35]: cool!

**Pull request merged**